recipe/meta.yaml

```diff
@@ -43,41 +43,41 @@ requirements:
   run:
     # appnope should be in orange-widget-base
     - appnope  # [osx]
-    - anyqt >=0.0.13
+    - anyqt >=0.1.0
     - baycomp >=1.0.2
-    - bottleneck >=1.3.2
+    - bottleneck >=1.3.4
     - chardet >=3.0.2
     - httpx >=0.21
-    - joblib >=0.9.4
+    - joblib >=0.11
     - keyring
     - keyrings.alt
-    - matplotlib-base >=2.2.5
+    - matplotlib-base >=3.2.0
     - networkx
-    - numpy >=1.17.3
+    - numpy >=1.19.5
     - openpyxl
     - opentsne >=0.6.1
-    - orange-canvas-core >=0.1.24,<0.2a
-    - orange-widget-base >=4.16.1
-    - pandas >=1.3.0
+    - orange-canvas-core >=0.1.28,<0.2a
+    - orange-widget-base >=4.19.0
+    - pandas >=1.3.0,!=1.5.0
     - pip >=18.0
     - pygments >=2.8.0
-    - pyqt >=5.12,!=5.15.1
-    - pyqtgraph >=0.11.1,!=0.12.4
+    - pyqt >5.15.1
+    - pyqtgraph >=0.12.2,!=0.12.4
     - pyqtwebengine >=5.12
     - python
     - python-louvain >=0.13
     - pyyaml
     - qtconsole >=4.7.2
     - requests
-    - scikit-learn >=1.0.1
-    - scipy >=1.3.1
+    - scikit-learn >=1.0.1,<1.2.0
+    - scipy >=1.9
     - serverfiles
     - setuptools >=41.0.0
     # xgboost is optional since user must install libomp (with brew) on macOS for xgboost to work
     # it is not required for xgboost from conda - it is why I am adding a package to the recipe
     - xgboost
     - catboost !=1.0.0  # [not linux or x86_64]
```

Comment:
@boldorider4 Could you please remove dependencies that are not required for runtime? 
See:
- https://github.com/biolab/orange3/blob/c48441485cce208cc8055fd13f7a1b95040a3066/setup.py#L84C43-L92
- https://github.com/biolab/orange3/blob/3.34.0/requirements-core.txt
- https://github.com/biolab/orange3/blob/3.34.0/requirements-gui.txt

I think it will unblock you

Reply:
done!

Reply:
`xgboost` and `catboost` are optional. They pollute the dependency tree, just remove them to simplify the build

Reply:
according to @pseudoyim we should keep optional dependencies if they don't cause issues as they'll facilitate the end user/dev not having to run extra environment setup errands.

Reply:
Ah, ok